mediagrains/grain.py

```diff
@@ -347,33 +362,19 @@ def normalise_time(self, value: Timestamp) -> Timestamp:
 
     @property
     def sync_timestamp(self) -> Timestamp:
-        ts = self.meta['grain']['sync_timestamp']
-        if isinstance(ts, str):
-            return Timestamp.from_tai_sec_nsec(ts)
-        else:
-            return ts
```

Comment:
The comment in https://github.com/bbc/rd-apmm-python-lib-mediagrains/blob/jamesba-timestampautoconvert/mediagrains/grain_constructors.py#L963 needs to be updated to say the timestamps must be strings only.

For some reason the `origin_timestamp` diverged from `sync/creation_timestamp` in https://github.com/bbc/rd-apmm-python-lib-mediagrains/commit/129b4682ce8715a48e6142fbf9aedcc6d5a9e19b.

Reply:
I decided instead to make it perform stringification during grain construction.